### PR TITLE
`stubDebug` diagnostic output

### PR DESCRIPTION
**Pull request by mernst**

### CODE REVIEW

File: framework/src/main/java/org/checkerframework/framework/stub/AnnotationFileParser.java [hunk 1]
```diff
@@ -712,6 +713,11 @@ public static void parseJdkFileAsStub(
       AnnotatedTypeFactory atypeFactory,
       ProcessingEnvironment processingEnv,
       AnnotationFileAnnotations stubAnnos) {
+    if (processingEnv.getOptions().containsKey("stubDebug")) {
+      System.out.printf(
```
Comment (smillst) [inline framework/src/main/java/org/checkerframework/framework/stub/AnnotationFileParser.java:717]: Is there a reason this is using `System.out.printf` and the other stubDebug methods are using  `.getMessager().printMessage(...)`?

**Review by smillst — CHANGES_REQUESTED** [review]

File: framework/src/main/java/org/checkerframework/framework/stub/AnnotationFileParser.java (hunk 1 above)
Comment (mernst) [reply framework/src/main/java/org/checkerframework/framework/stub/AnnotationFileParser.java:719]: I added a comment to explain.

Comment (smillst) [reply framework/src/main/java/org/checkerframework/framework/stub/AnnotationFileParser.java:719]: What about using `stubDebugStatic`?

Comment (mernst) [reply framework/src/main/java/org/checkerframework/framework/stub/AnnotationFileParser.java]: That is better!

**Review by smillst — APPROVED** [review]

**Review by smillst — APPROVED** [review]

**Pull request merged**